CRT.c

```diff
@@ -1046,7 +1046,7 @@ void CRT_init(const Settings* settings, bool allowUnicode, bool retainScreenOnEx
       CRT_scrollHAmount = 5;
    }
 
-   if (termType && (String_startsWith(termType, "xterm") || String_eq(termType, "vt220"))) {
+   if (termType && (String_startsWith(termType, "xterm")  || String_eq(termType, "vt220") || String_startsWith(termType, "screen") || String_startsWith(termType, "tmux") || String_startsWith(termType, "st"))) {
```

Comment:
If there are other `$TERM` values that could also benefit from being here, I would appreciate it if they were pointed out.